docs/sources/upgrading/_index.md

```diff
@@ -85,6 +85,17 @@ The global `deletion_mode` option in the compactor configuration moved to runtim
 - The `deletion_mode` global override needs to be set to the desired mode: `disabled`, `filter-only`, or `filter-and-delete`. By default, `filter-and-delete` is enabled.
 - Any `allow_delete` per-tenant overrides need to be removed or changed to `deletion_mode` overrides with the desired mode.
 
+### Promtail config change
+
+add `max_streams` config in Promtail .
+```yaml
+
+limits_config:
+  readline_rate: 100
+  readline_burst: 200
+  max_streams: 100
+```
+
```

Comment:
```suggestion
```
Hey since the way you added `max_streams` is backwards-compatible, I think you can leave it out of the upgrading guide.